app/graph/mutations/feed_invitation_mutations.rb

```diff
@@ -0,0 +1,44 @@
+module FeedInvitationMutations
+  MUTATION_TARGET = 'feed_invitation'.freeze
+  PARENTS = ['feed'].freeze
+
+  class Create < Mutations::CreateMutation
+    argument :email, GraphQL::Types::String, required: true
+    argument :feed_id, GraphQL::Types::Int, required: true, camelize: false
+  end
+
+  class Destroy < Mutations::DestroyMutation; end
+
+  class Accept < Mutations::UpdateMutation
+    argument :id, GraphQL::Types::Int, required: true
+    argument :team_id, GraphQL::Types::Int, required: true, camelize: false
+
+    field :success, GraphQL::Types::Boolean, null: true
+
+    def resolve(id: nil, team_id: nil)
```

Comment:
I remember for required fields the method declaration is  `def resolve(id:, team_id:)`

Reply:
Fixed!